test/e2e/empty_dir_wrapper.go

```diff
@@ -0,0 +1,201 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/util"
+	"k8s.io/kubernetes/pkg/util/intstr"
+
+	"strconv"
+
+	. "github.com/onsi/ginkgo"
+)
+
+// This test will create a pod with a secret volume and gitRepo volume
+// Thus requests a secret, a git server pod, and a git server service
+var _ = Describe("EmptyDir wrapper volumes", func() {
+	f := NewFramework("secrets")
+
+	It("should becomes running [Conformance]", func() {
```

Comment:
@resouer Why don't you mark this skipped with `[Skipped]` and we'll chew on how to cut the image size down in a follow-up?

cc @thockin